server/views/pages/overview.njk

```diff
@@ -546,3 +546,7 @@
     </div>
   </div>
 {% endblock %}
+{% block bodyEnd %}
+  {{ super() }}
+  <script src="/assets/js/disable-on-submit.js"></script>
```

Comment:
Not for now, but I noticed the moj button macro has a double click prevention thing built in. Maybe we can get rid of this in favour of that in the future

Reply:
That would be nice - I'll create a tech debt ticket to sort it out.